scripts/pynag

```diff
@@ -690,6 +700,45 @@ def pretty_print(objects, attributes=None):
         pre = "-"*10
         post = "-"*(80-10-len(message))
         print pre + message + post
+
+def get_nagios_attributes(obj):
+    obj_attributes = {}
+    for attribute, value in obj.items():
+        # Don't return pynag internal attributes
+        # except the object type
+        if attribute == "id":
+            continue
+        if attribute == "meta":
+            obj_attributes["object_type"] = value['object_type']
```

Comment:
Agreed. Should we just dump meta as it is or have object_type in the top level? Should we just dump the object as it is?